### PR TITLE
Fix pilot post submit (through restoring pkg_coverage.sh)

### PR DESCRIPTION
**Pull request by yutongz**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #https://github.com/istio/istio/issues/631

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by ldemailly** [comment]
this needs to be rebased yet the UI is gone, probably the repo settings have been changed again

**Comment by ldemailly** [comment]
thx for rebase, let see if we're more lucky on the 2nd test run

File: codecov.report [hunk 1]
```diff
@@ -0,0 +1,14 @@
+ok  	istio.io/pilot/adapter/config/aggregate	0.291s	coverage: 92.6% of statements
```
Comment (ldemailly) [inline codecov.report]: I don't think this should be checked in ?

**Comment by ldemailly** [comment]
can we stick to making postsubmit work (btw how is it tested?) and deal with the coverage separately ?

File: codecov.report (hunk 1 above)
Comment (yutongz) [reply codecov.report]: Thanks!

**Comment by yutongz** [comment]
Disabled in postsubmit. Separated job (pilot-coverage) is not reported.

**Comment by yutongz** [comment]
It seems pilot-presubmit flaky?

**Comment by ldemailly** [comment]
@kyessenov the pres submit passes, the e2esmoke error is unrelated, can u merge this to see if we get postsubmit working again? tia

**Comment by ldemailly** [comment]
@andraxylia @rshriram @kyessenov 
this PR needs to be merged. we need to build artefacts in postsubmit asap

/retest

**Comment by ldemailly** [comment]
as you can see with https://github.com/istio/pilot/pull/1172 master doesn't pass the smoke test either so can someone please merge this (or give me admin and I will)

**Comment by ldemailly** [comment]
Thanks ! Hope it works

**Pull request merged**